tests/h/streamer/filter_test.py

```diff
@@ -25,41 +25,50 @@ class TestFilterHandler:
             (["http://example.com"], "https://example.com/?", True),
         ],
     )
-    def test_it_matches_uri(self, query_uris, ann_uri, should_match):
+    def test_it_matches_uri(self, factories, query_uris, ann_uri, should_match):
         query = {
             "match_policy": "include_any",
             "actions": {},
             "clauses": [{"field": "/uri", "operator": "one_of", "value": query_uris}],
         }
         handler = FilterHandler(query)
 
-        ann = {"id": "123", "uri": ann_uri}
+        ann = factories.Annotation(target_uri=ann_uri)
         assert handler.match(ann) is should_match
 
-    def test_it_matches_id(self):
+    def test_it_matches_id(self, factories):
+        ann_a = factories.Annotation(target_uri="https://example.com")
+        ann_b = factories.Annotation(target_uri="https://example.net")
```

Comment:
I think it would be nice to state our intention here e.g.
```suggestion
        ann_matching = factories.Annotation(target_uri="https://example.com")
        ann_non_matching = factories.Annotation(target_uri="https://example.net")
```

Reply:
Seems reasonable. Will do.